contrib/kiss-depends

```diff
@@ -1,9 +1,15 @@
 #!/bin/sh -ef
 # Display a package's dependencies
 
+# CRUX style usage using the current directory as the name of the package to be
+# operated on.
```

Comment:
For people who aren't familiar with CRUX, and to fit on one line, how about
`# Use the current directory as the package name if no package is given`

Reply:
I copied the comment from the kiss script. This sounds better indeed, I'll also change it in the kiss script then I think.